src/elements/content-sidebar/activity-feed/annotations/AnnotationActivityMenu.js

```diff
@@ -89,4 +100,5 @@ const AnnotationActivityMenu = ({ canDelete, canEdit, id, onDeleteConfirm, onEdi
     );
 };
 
-export default AnnotationActivityMenu;
+export { AnnotationActivityMenu as AnnotationActivityMenuBase };
+export default flow([withFeatureConsumer])(AnnotationActivityMenu);
```

Comment:
Why do we need `flow` for only one function?

Reply:
Good call, I don't think we need it.